### DISCUSSION TITLE
Design Token Request: colorBackgroundWeak

### DISCUSSION THREAD

**Discussion by mannehedlund** [twilio-labs/paste#2114] [Ideas] [answered]
Hello,

I'm using a Box to make a line (not using Separator component for unrelated issues). I'm currently using `colorBackgroundStronger` token for this (gray-50), whereas my design calls for gray-30 such as `colorBorderWeak`. Something like a `colorBackgroundWeak` design token would be perfect -- perhaps others would benefit from this as well.

Here is my desired result (no background color design token for it).
![image](https://user-images.githubusercontent.com/46298027/148258057-1be573e4-474e-4d6c-b474-e7b30a9e34e3.png)

And here is my current result.
![image](https://user-images.githubusercontent.com/46298027/148258127-1f350958-d698-456e-8396-7956bbfd7bc8.png)

**Comment by shleewhite** [comment, marked as answer]
Hi @mannehedlund! Does the toke `colorBackgroundStrong` work for your use case? It is gray-20.

**Reply by mannehedlund** [reply]
It looks _okay_, so does `colorBackgroundStronger` (a bit better imo), but both are compromising the designs (gray-30).

**Reply by shleewhite** [reply]
Following up from office hours today, I remember you saying that you couldn’t use `<Separator>` because it was the wrong color. [Looking at the Separator code](https://github.com/twilio-labs/paste/blob/a7d1d0b7456a2014c18db520ff31177085ba371b/packages/paste-core/components/separator/src/index.tsx#L31), it seems like it does use colorBorderWeak which is `gray-30` .

If that doesn't work, we would actually prefer you to use `<Box>` to create your own separator like this (should also fix the vertical alignment problem):
```
<Box
  as="hr"
  aria-orientation="horizontal"
  margin="space0"
  borderColor="colorBorderWeak" //or any other border token
  borderStyle="solid"
  borderWidth="borderWidth0"
  borderBottomWidth="borderWidth10"
  width="auto"
/>
```

Thanks for coming to talk with us today, please let me know if you have any additional questions!

**Reply by mannehedlund** [reply]
Yeah, Paste Separator actually worked for achieving the above desired result. However, I will also need identical separators that are red, which is where the Paste Separator no longer works, as the border is hard-coded. I should've clarified that hard-coded color didn't work -- my bad. The code you gave is essentially what I used, except the `height=1px` instead of `borderBottom`. The 1px off center is still an issue, but I can probably find a workaround.

Does this mean that you will not be adding a custom color option to the Separator component?

**Reply by shleewhite** [reply]
Ahhh, I see. Yeah, we talked about it further after office hours and decided to hold off because we are not sure it is appropriate for all use cases.

For the text being 1px off center, can you send a screenshot of what it looks like?

**Reply by mannehedlund** [reply]
You know what, you're right, it looks the same. Did not consider `box-sizing: border-box` which your Box component is. It is center and the right color now using the border. Thanks for the help! :)

**Reply by shleewhite** [reply]
Woo! Glad we could come up with a solution!
